docs/03-kb.md

```diff
@@ -170,3 +170,20 @@ You can summarize one resource or a batch of resources. It will produce a summar
   kb = sdk.NucliaKB()
   kb.summarize(resources=["RID,RID1,RID2,RID3"])
   ```
+
+## Logs
+
+You can get the logs of a Knowledge Box. There are different types of logs: NEW, PROCESSED, MODIFIED, CHAT, SEARCH, FEEDBACK
```

Comment:
maybe include an explanation of what each type of log really means? Or a pointer to some other doc where it is explained?